api/types/system_role.go

```diff
@@ -105,6 +105,21 @@ var roleMappings = map[string]SystemRole{
 	"mdm":             RoleMDM,
 }
 
+func normalizedSystemRole(s string) SystemRole {
+	if role, ok := roleMappings[strings.ToLower(strings.TrimSpace(s))]; ok {
+		return role
+	}
+	return SystemRole(s)
```

Comment:
As I understand that's just not to have an error for ease of usage and idea is it will error out upper?